README.md

```diff
@@ -4,14 +4,47 @@
 
 A library for adding reform standard compliant healthchecks to nodejs applications.
 
+It exposes 3 endpoints:
+
+1. `/health` - Returns 200 by default  along with `buildInfo`, can optionally include result evaluating all `checks` passed in config.
+2. `/health/livness` - Returns 200 always.
+3. `/health/readiness` - Returns 200 by default , can optionally include result evaluating all `readinessChecks` passed in config.
+
 ## Usage
 
 Configure an express.js handler with checks.
 
 ```javascript
 const healthcheck = require('@hmcts/nodejs-healthcheck');
 
-app.get("/status", healthcheck.configure({
+const config = {
+  checks: {
+    mySimpleWebCheck: healthcheck.web("https://example.com/status"),
+    myComplexWebCheck: healthcheck.web("https://example.com/other", {
+      callback: (err, res) => {
+        return res.body.status == "good" ? healthcheck.up() : healthcheck.down()
+      },
+      timeout: 5000,
+      deadline: 10000,
+    }),
+    myRawCheck: healthcheck.raw(() => {
+      return myInternalCheck() ? healthcheck.up() : healthcheck.down()
+    })
+  },
+  buildInfo: {
+    myCustomBuildInfo: "yay"
+  }
+};
+
+ healthcheck.addTo(app, config);
+```
+
+You can optionally include some readiness checks.
```

Comment:
Guidance on what you should include and not include seems to be missing, I would use redis as an example